Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imdining",category=['you'],prompt="I'm Dining",pool=True,unlocked=True))
+if seen_event(monika_imdining) = False:
+    label monika_imdining:
```

Comment:
I see that all 4 of these lines have the same open mouth. Is Monika waiting for the player to feed her something? ~~Me too... me too...~~

They're all almost the exact same expression. I would vary them up a little.